src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -521,21 +521,27 @@ private function setMaxSteps($max)
      */
     private function overwrite($message)
     {
+        static $firstTime = true;
```

Comment:
Having a private property is fine.

The pain is that this is going to be a test every time the progress bar is updated.

Somewhere there has to be a switch between 1st and not 1st, so I suspect there is no way to not have the test.